### PR TITLE
Hermes: Add prv publish time metadata

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: target_chains/sui/sdk/js/src/examples/SuiRelay.ts [hunk 1]
```diff
@@ -18,9 +18,8 @@ const argvPromise = yargs(hideBin(process.argv))
     type: "array",
     demandOption: true,
   })
-  .option("price-service", {
```
Comment (ali-bahjati) [inline target_chains/sui/sdk/js/src/examples/SuiRelay.ts:21]: this is a minor change to move to hermes

File: hermes/src/aggregate.rs [hunk 2]
```diff
@@ -333,7 +334,7 @@ where
         .iter()
         .map(|message_state| match message_state.message {
             Message::PriceFeedMessage(price_feed) => Ok(PriceFeedUpdate {
-                price_feed:  PriceFeed::new(
+                price_feed:        PriceFeed::new(
```
Comment (cctdaniel) [inline hermes/src/aggregate.rs:337]: the extra spacing seems weird

**Review by cctdaniel — APPROVED** [review]

File: hermes/src/aggregate.rs (hunk 2 above)
Comment (ali-bahjati) [reply hermes/src/aggregate.rs:337]: yeah it is the rust formatter config that we have :D

File: hermes/src/aggregate.rs [hunk 3]
```diff
@@ -152,10 +152,11 @@ pub enum Update {
...
 pub struct PriceFeedUpdate {
-    pub price_feed:  PriceFeed,
-    pub slot:        Option<Slot>,
-    pub received_at: Option<UnixTimestamp>,
-    pub update_data: Option<Vec<u8>>,
+    pub price_feed:        PriceFeed,
+    pub slot:              Option<Slot>,
+    pub received_at:       Option<UnixTimestamp>,
+    pub update_data:       Option<Vec<u8>>,
+    pub prev_publish_time: Option<UnixTimestamp>,
```
Comment (Reisen) [inline hermes/src/aggregate.rs:159]: I realise we can't change it at this point but It's a shame this isn't `last_publish_time` like other `last_` fields we have in our codebases.

File: price_service/client/js/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/price-service-client",
-  "version": "1.7.0",
+  "version": "1.8.0",
```
Comment (Reisen) [inline price_service/client/js/package.json:3]: 👌 on the respecting semver

**Review by Reisen — APPROVED** [review]

File: hermes/src/aggregate.rs (hunk 3 above)
Comment (ali-bahjati) [reply hermes/src/aggregate.rs:159]: i think here `prev` is more clear than `last`.

**Pull request merged**